tests/identities/TestProvider.ts

```diff
@@ -7,17 +7,15 @@ import type {
 } from '@/identities/types';
 import type { Claim } from '@/claims/types';
 import type { IdentityClaim, IdentityClaimId } from '@/identities/types';
-import type { NodeId } from '@/nodes/types';
 
 import { Provider, errors as identitiesErrors } from '@/identities';
-import { createClaim, decodeClaim } from '@/claims/utils';
 
 class TestProvider extends Provider {
   public readonly id = 'test-provider' as ProviderId;
 
-  protected linkIdCounter: number = 0;
+  public linkIdCounter: number = 0;
   protected users: Record<IdentityId | string, POJO>; // FIXME: the string union on VaultId is to prevent some false errors.
-  protected links: Record<IdentityClaimId | string, string>; // FIXME: the string union on VaultId is to prevent some false errors.
+  public links: Record<IdentityClaimId | string, string>; // FIXME: the string union on VaultId is to prevent some false errors.
   protected userLinks: Record<
```

Comment:
Why not make `userLinks` and `userTokens` also public so they can be later mutated too.

Reply:
`userTokens` and `userLinks` are both very hard-coded into some of the functions in the test provider, and changing them can break the entire thing. I think it's safer to leave as is but I can make them public if you'd prefer